look-and-feel/react/src/Form/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/react";
+import { TextArea } from "../TextArea";
+
+describe("<TextArea />", () => {
+  test("should render component with default props", () => {
+    render(
+      <TextArea
+        label="Label"
+        value="Lorem Ipsum"
+        description="Description"
+        helper="Helper"
+        buttonLabel="Button label"
+        error="Error message"
+      />,
+    );
+
+    screen.getByRole("textbox");
+    screen.getByText("Label");
+    screen.getByText("Lorem Ipsum");
+    screen.getByText("Description");
+    screen.getByText("Helper");
+    screen.getByRole("button", { name: "Button label" });
+    screen.getByText("Error message");
+  });
+
+  test("should render asterisk if required", () => {
+    render(<TextArea label="Required label" required />);
+
+    screen.getByText("Required label");
+    screen.getByText("*");
```

Comment:
Tu ne peux pas juste tester ça ?
```suggestion
    screen.getByText("Required label *");
```

Reply:
L'astérisque est dans un span à part, je suis obligé de tester les deux séparément

Reply:
Le span est vraiment nécessaire ?